src/cmd/cli/command/commands.go

```diff
@@ -84,7 +84,7 @@ func Execute(ctx context.Context) error {
 		}
 
 		if err.Error() == "resource_exhausted: maximum number of projects reached" {
-			printDefangHint("To deactivate a project, do:", "compose down")
+			printDefangHint("To deactivate a project, do:", "compose down --project-name <name>")
```

Comment:
I like this. Wonder if we should bother getting the existing project's name (currently always 1) and show that in the hint?

Reply:
I'm tempted to push leave it like this for the moment, because this PR has already grown quite large, and I still need to get familiar with the best way to get the list of project names from fabric. But I agree this should be the next step.